### DISCUSSION TITLE
No Fleet Server or Unhealthy Fleet Server

### DISCUSSION THREAD

**Discussion by Phillip-Samson** [Security-Onion-Solutions/securityonion#10614] [2.4]
Oracle Rocky 9.2 Beta 3 Standalone
If Install standalone first, I always end up with Fleet Server unconfigured.
However, If I install the Eval version first the fleet server is running great. but my Darktrace or SentinelOne Integrations won't store logs. I believe this because Logstash is not configured.

After I install the Eval version, I then install the Standalone. Everything seems to install fine with no errors but the Fleet Server goes offline. 

<img width="1216" alt="image" src="https://github.com/Security-Onion-Solutions/securityonion/assets/36277967/4ddd43e5-2a3a-4c88-a47b-0a31ec883cc1">

When I log into the Elastic Ageb docker container I see that the elastic agent is not running.
elastic-agent@FleetServer-orseconp02:~$ elastic-agent status
State: DEGRADED
Message: 1 or more components/units in a degraded state
Fleet State: STARTING
Fleet Message: (no message)
Components:
  * fleet-server  (HEALTHY)
                  Healthy: communicating with pid '21'

elastic-agent@FleetServer-orseconp02:~$ elastic-agent inspect
Error: no fleet config retrieved yet
For help, please see our troubleshooting guide at https://www.elastic.co/guide/en/fleet/8.7/fleet-troubleshooting.html

How should I approach this?

**Comment by dougburks** [comment]
Please try Beta 4 and see if that helps:
https://blog.securityonion.net/2023/07/security-onion-24-beta-4-release-now.html

**Comment by Phillip-Samson** [comment]
That worked
